code/modules/cargo/bounties/chef.dm

```diff
@@ -136,3 +136,8 @@
 	required_count = 6
 	wanted_types = list(/obj/item/reagent_containers/food/snacks/nugget)
 
+/datum/bounty/item/chef/englishbreakfast
```

Comment:
Please follow the naming scheme

Reply:
I believe this follows the naming scheme

Reply:
if its about the word "english", then note that we literally have a cuban carp bounty

Reply:
I meant naming it english_breakfast

Reply:
Alright, gonna change that tomorrow once I get some sleep